finch/processes/wps_xsubset_point_dataset.py

```diff
@@ -94,11 +92,16 @@ def _handler(self, request: WPSRequest, response: ExecuteResponse):
 
         variable = request.inputs["variable"][0].data
         variables = None if variable is None else [variable]
-        rcp = single_input_or_none(request.inputs, "rcp")
+        scenario = single_input_or_none(request.inputs, "scenario")
+        models = [m.data.strip() for m in request.inputs["models"]]
 
-        dataset_name = single_input_or_none(request.inputs, "dataset_name")
+        dataset_name = single_input_or_none(request.inputs, "dataset")
+        if dataset_name == 'bccaqv2':
+            dataset_name = 'candcs-u5'
```

Comment:
@aulemahal Do we plan to support this conversion in the long run, or is it temporary?

Reply:
I'm not sure...I'd say we keep it only temporarily, but I don't know who the users are really. If on your side, you never need the fix, I'll just remove in the next release.

The "problem" is that I can't make a "FutureWarning" or similar, so it will simply break for the users (I'm not sure there are many, though).

Reply:
Do you think that supporting the conversion will need other modification than the ones done in this PR?

Reply:
Why not just make a copy of the yaml config for candcs-u5 called bccaqv2. You would not have to special case anything.

Reply:
yo.
Hacky config instead of hacky code? I'm in.